buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/PluginZipPath.kt

```diff
@@ -0,0 +1,8 @@
+package com.virtuslab.gitmachete.buildsrc
+
+import org.gradle.api.Project
+
+fun Project.printPluginZipPath() {
+  val buildPlugin = tasks.findByPath(":buildPlugin")!!
+  println(buildPlugin.outputs.files.first().path)
```

Comment:
I don't think this function merits a a separate file... in fact these 2 lines of body could just go directly into `doLast { ... }` block

Reply:
sure, done